apps/council-ui/src/ui/voters/VotersListCompact.tsx

```diff
@@ -1,45 +1,79 @@
-import { Voter } from "@council/sdk";
-import Link from "next/link";
-import { ReactElement } from "react";
+import { VoterWithPower } from "@council/sdk";
+import { ReactElement, useMemo, useState } from "react";
 import { makeVoterURL } from "src/routes";
+import { formatBalance } from "src/ui/base/formatting/formatBalance";
+import {
+  SortableGridTable,
+  SortOptions,
+} from "src/ui/base/tables/SortableGridTable";
 import { useBulkEnsRecords } from "src/ui/ens/useBulkEnsRecords";
 import { VoterAddress } from "src/ui/voters/VoterAddress";
 
 interface VotersListCompactProps {
-  voters: Voter[];
+  voters: VoterWithPower[];
 }
 
 export function VotersListCompact({
   voters,
 }: VotersListCompactProps): ReactElement {
+  const [sortOptions, setSortOptions] = useState<SortOptions<"votingPower">>({
+    key: "votingPower",
+    direction: "DESC",
+  });
+
+  const sortedVoters = useMemo(() => {
+    let sorted = voters;
+    const { key, direction } = sortOptions;
+
+    if (direction) {
+      switch (key) {
+        case "votingPower":
+        default:
+          sorted = voters
+            .slice()
+            .sort((a, b) => +a.votingPower - +b.votingPower);
+          break;
+      }
+
+      if (direction === "DESC") {
+        sorted.reverse();
+      }
+    }
+
+    return sorted;
+  }, [sortOptions, voters]);
+
   const { data: ensRecords } = useBulkEnsRecords(
-    voters.map((voter) => voter.address),
+    voters.map(({ voter }) => voter.address),
   );
 
   return (
-    <table className="w-full daisy-table daisy-table-compact">
-      <thead>
-        <tr>
-          <th>Voter</th>
-        </tr>
-      </thead>
-      <tbody>
-        {voters.map((voter) => {
-          const ensName = ensRecords?.[voter.address];
-          return (
-            <tr key={voter.address}>
-              <th>
-                <Link
-                  href={makeVoterURL(voter.address)}
-                  className="flex items-center gap-x-2 hover:underline"
-                >
-                  <VoterAddress address={voter.address} ensName={ensName} />
-                </Link>
-              </th>
-            </tr>
-          );
-        })}
-      </tbody>
-    </table>
+    <SortableGridTable
+      headingRowClassName="[&>*]:px-4 [&>*]:py-2"
+      bodyRowClassName="[&>*]:px-4 [&>*]:py-2 text-sm font-semibold"
+      zebra={false}
```

Comment:
nit: I feel like this should be called `striped` for clarity. Is this a common name in other systems?